src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -186,14 +186,20 @@ Supported in Sentry Android Gradle Plugin version `3.0.0` and above.
 
 </Note>
 
-The [Sentry Android Gradle Plugin](/platforms/android/proguard/#gradle-configuration) does the tracing auto instrumentation via bytecode manipulation for `SQLite` and `Room` libraries.
+The [Sentry Android Gradle Plugin](/platforms/android/proguard/#gradle-configuration) does the tracing auto instrumentation via bytecode manipulation for `androidx.sqlite` and `androidx.room` libraries.
 
 The Plugin injects a code snippet that starts a span out of the active span bound to the scope for each `CRUD` operation. The SDK sets the span `operation` to `db` and `description` to the SQL Query if available.
 
 The span finishes once the operation has been executed. The span `status` is set to `SpanStatus.OK` if successful or `SpanStatus.INTERNAL_ERROR` if there was any error.
 
 When the operation throws an `Exception`, Sentry's SDK associates this exception to the running span. If you haven't set the SDK to swallow the exception and capture it, the span and SentryEvent will be linked when viewing it on the **Issue Details** page in sentry.io.
 
+<Note>
+ 
+At the moment, only a standard `androidx.room` usage is supported, i.e. SQL queries for any other `SupportSQLiteOpenHelper.Factory` than [androidx.sqlite](https://github.com/androidx/androidx/tree/androidx-main/sqlite) will not be reported. However, it's worth reporting an [issue](https://github.com/getsentry/sentry-android-gradle-plugin/issues), as we could consider adding a new integration.
```

Comment:
I don't fully understand what you mean by

> However, it's worth reporting an issue, as we could consider adding a new integration.

For what should I report an issue?

Reply:
Hm, do you understand the previous sentence?

Reply:
I'm also struggling with this last sentence. We've told the users about a limitation. Is this just a general appeal for people to tell us about problems with this plugin? Maybe this would be a better approach?
```suggestion
At the moment, only a standard `androidx.room` usage is supported, i.e. SQL queries for any other `SupportSQLiteOpenHelper.Factory` than [androidx.sqlite](https://github.com/androidx/androidx/tree/androidx-main/sqlite) will not be reported. If you experience any problems using this plugin, please report the [issue](https://github.com/getsentry/sentry-android-gradle-plugin/issues), as we might consider adding a new integration.
```

Reply:
Alright, my wording is bs then. What I meant is not a general appeal, but something like "If you are using a different `SupportSQLiteOpenHelper.Factory`, please report an issue on Github, so we are aware and can possibly tackle this"